spec/unit/pops/evaluator/access_ops_spec.rb

```diff
@@ -24,115 +24,115 @@ def binary(s)
     # Note that the factory is not aware of Binary and cannot operate on a
     # literal binary. Instead, it must create a call to Binary.new() with the base64 encoded
     # string as an argument
-    CALL_NAMED(QREF("Binary"), true, [Base64.strict_encode64(s)])
+    CALL_NAMED(QREF("Binary"), true, [infer(Base64.strict_encode64(s))])
```

Comment:
Because the expression needs a LiteralExpression, not a String. The Factory infers an expression, not a Type.